packages/datadog-instrumentations/src/rhea.js

```diff
@@ -45,7 +45,9 @@ addHook({ name: 'rhea', versions: ['>=1'], file: 'lib/link.js' }, obj => {
     const { host, port } = getHostAndPort(this.connection)
 
     const targetAddress = this.options && this.options.target &&
-      this.options.target.address ? this.options.target.address : undefined
+      this.options.target.address
+      ? this.options.target.address
+      : undefined
```

Comment:
shouldn't it be 
```suggestion
      this.options.target.address
        ? this.options.target.address
        : undefined
```

Reply:
The linter auto-fixed it like that. Probably just not smart enough to parse the indentation correctly.